var/spack/repos/builtin/packages/py-chainer/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
+
+from spack import *
+
+
+class PyChainer(PythonPackage):
+    """
+    Chainer is a Python-based deep learning framework aiming at flexibility.
+
+    It provides automatic differentiation APIs based on the define-by-run
+    approach (a.k.a. dynamic computational graphs) as well as object-oriented
+    high-level APIs to build and train neural networks.
+    It also supports CUDA/cuDNN using CuPy for high performance training
+    and inference.
+    """
+
+    homepage = "https://chainer.org/"
+    url      = "https://github.com/chainer/chainer/archive/v7.2.0.tar.gz"
```

Comment:
You can add me as a maintainer for this package:
```python
maintainers = ['adamjstewart']
```
I maintain all of the other DL packages in Spack, so why not 1 more?